code/modules/hydroponics/compost_bin.dm

```diff
@@ -0,0 +1,195 @@
+#define BASE_BIOMASS_CAPACITY 1500
+#define BASE_COMPOST_CAPACITY 1500
+#define SOIL_COST 25
+#define SOIL /obj/item/stack/sheet/soil
```

Comment:
I dont think that SOIL needs to be a define like this

Reply:
Just use the path as is?